docs/en/sql-reference/dictionaries/external-dictionaries/external-dicts-dict-lifetime.md

```diff
@@ -12,6 +12,11 @@ Dictionary updates (other than loading for first use) do not block queries. Duri
 
 Example of settings:
 
+:::tip
+If you are using a DICTIONARY with ClickHouse Cloud please use the DDL query option to create your dictionaries,  and create your dictionary as user `default`.
```

Comment:
@DanRoscigno why DICTIONARY is in caps lock here and in similar places below?

Reply:
Also probably deserves an include instead of copy-paste.

Reply:
moved to snippet file and changed case
![image](https://user-images.githubusercontent.com/25182304/201989757-67aa3092-7996-4968-a38a-dd29f91eaffd.png)